app/core/tests/test_api.py

```diff
@@ -156,3 +156,17 @@ def test_create_sponsor_partner(auth_client):
     }
     res = auth_client.post(SPONSOR_PARTNERS_URL, payload)
     assert res.status_code == status.HTTP_201_CREATED
+
+
+def test_create_faq(admin_client):
+
+    payload = {
+        "question": "How do I work on an issue",
+        "answer": "See CONTRIBUTING.md",
+        "tool_tip_name": "How to work on an issue",
+    }
+    res = admin_client.post(FAQ_URL, payload)
+    assert res.status_code == status.HTTP_201_CREATED
+
+    res = admin_client.get(FAQ_URL, payload)
+    assert res.status_code == status.HTTP_200_OK
```

Comment:
Strictly speaking, this should be in it's own test like `test_get_faqs()`. I think it returns an array even if there's only 1 item. That is if you wanted to check the response data.

Extra: If you wanted to test the single item endpoints like GET, PUT, PATCH, DELETE. You can declare 
`faq_url = reverse("faq-detail", args=[faq.uuid])` and use it with those action function in `admin_client` or `auth_client`.